Dockerfile-debug

```diff
@@ -1,9 +1,20 @@
-FROM gcr.io/linkerd-io/base:2020-06-08.01
+FROM debian:buster-20200514-slim
 RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
+    dnsutils \
+    iptables \
+    jq \
+    nghttp2 \
     tcpdump \
     iproute2 \
     lsof \
     conntrack \
     tshark && \
     rm -rf /var/lib/apt/lists/*
+
+# Switching base image from stretch to buster, we need to use iptables-legacy
+# if no changes to how iptables is invoked are done.
```

Comment:
Can you please remove this comment? No need to explain the details about the stretch-to-buster upgrade, plus the second sentence doesn't make sense. It can be replaced with something like  "We still rely on old iptables-legacy syntax"
(Same thing for `cni-plugin/Dockerfile`)